include/zephyr/bluetooth/conn.h

```diff
@@ -252,6 +252,93 @@ struct bt_conn_le_subrate_changed {
 	uint16_t supervision_timeout;
 };
 
+/** Most Connection Interval Groups possible
+ *
+ * The practical maximum is 41 groups based on HCI event size constraints:
+ *     (HCI event max size - cmd_complete overhead - response fields) / sizeof(group)
+ *     (255 - 4 - 3) / 6 = 41 groups max
+ */
+#define BT_CONN_LE_MAX_CONN_INTERVAL_GROUPS 41
+
+/** Minimum supported connection interval group
+ *
+ * Each group represents an arithmetic sequence of supported connection intervals:
+ *   min, min+stride, min+2*stride, ..., min+n*stride (where min+n*stride <= max)
```

Comment:
Suggest to be consistent with spaces

```suggestion
 *   min, min + stride, min + 2 * stride, ..., min + n * stride (where min + n * stride <= max)
```

or

```suggestion
 *   min, min+stride, min+2*stride, ..., min+n*stride (where min+n*stride<=max)
```

Not sure if we have any guidelines for this in the documentation though (we require spaces around operators in code).